### ISSUE TITLE
Date Picker: Narrator focus lands twice on same date button

### ISSUE THREAD

**Issue by ipip2005** [OfficeDev/office-ui-fabric-react#2166]
Note: I'm porting a bug from the a11y conformance testing for web parts reported by vendors. We triage this bug and decided to move it here and start a discussion.
Questions: What is the **screen reader support matrix for fabric-react** and how fabric-react will triage this kind of bugs? 

### Bug Report

- __Package version(s)__: latest
- __Browser and OS versions__:
Browser: Edge
OS build: 16192.rs_prerelease.170505-1540
Impacted Webparts: Events

#### Priorities and help requested (not applicable if asking question):

Requested priority: (Low) 

Products/sites affected: web parts

#### Describe the issue:

User Experience:
When screen reader user navigates on a date picker box using Caps+ arrow key and narrator reads incomplete information three times then user will get irritate and keyboard focus, and narrator focus does not move in sync will create more issue to screen reader users using Caps+ arrow key.
Actual:
When user press first time Caps+ arrow key, on suppose 28 Date then Narrator reads ‘May 28, 2017 non-selected, May 28, 2017 button non-selected’.
When user again press Caps+ arrow key, then narrator focus remains there, and keyboard focus come on it and narrator again reads ‘May 28, 2017 button’.
Using Caps+ arrow key, keyboard focus and narrator focus does not move in sync and does not read complete information as narrator reads using tab key.
 
Expected:
When user press Caps+ arrow key, then narrator should read complete information only once i.e.; ‘May 28, 2017 button column header Sunday non-selected’.

Note: On opening of calendar dialog, narrator reads 'Dialog calendar focus on june 22 2017 button group june 2017'. Here narrator reads extra irrelevant information 'group june 2017'.
 
MAS Reference:
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7BBF8AF12E-731E-4129-A439-077991F93C2C%7D&file=MAS%2033J%20%E2%80%93%20Tracking%20of%20Focus%20and%20Selection%20Attributes.docx&action=default

### If applicable, please provide a codepen repro:

https://dev.office.com/fabric#/components/datepicker

Video reproduces the issue: https://microsoftapc.sharepoint.com/portals/hub/_layouts/15/PointPublishing.aspx?app=video&p=p&chid=46c8c422-43de-4a55-9b05-f57696c91c7b&vid=9d459d50-dbca-40a4-a6c2-1d59d51729ee
